engine/src/main/java/org/hibernate/search/engine/common/execution/spi/SimpleScheduledExecutor.java

```diff
@@ -6,16 +6,22 @@
  */
 package org.hibernate.search.engine.common.execution.spi;
 
+import java.lang.invoke.MethodHandles;
+import java.util.Locale;
 import java.util.concurrent.Future;
 import java.util.concurrent.RejectedExecutionException;
 import java.util.concurrent.ScheduledFuture;
 import java.util.concurrent.TimeUnit;
 
+import org.hibernate.search.engine.logging.impl.Log;
 import org.hibernate.search.util.common.annotation.Incubating;
+import org.hibernate.search.util.common.logging.impl.LoggerFactory;
 
 @Incubating
 public interface SimpleScheduledExecutor {
 
+	Log log = LoggerFactory.make( Log.class, MethodHandles.lookup() );
```

Comment:
This public field effectively becomes SPI, and thus we're leaking `Log`, which is an internal class...

Since you're not using it anyway, please remove it?